.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,16 +12,16 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
+Steps to reproduce the behaviour:
```

Comment:
This is actually the spelling of behavior in standard American English, see [this](https://www.merriam-webster.com/dictionary/behavior). :smile:

Reply:
Oh sorry -- I did not know about this. Thanks! Closing this PR

Reply:
no worries, we have an international team :smile: